modules/ossm-about-migrating-to-cluster-wide.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/ossm-deployment-models.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-about-migrating-to-cluster-wide_{context}"]
+= About migrating to a cluster-wide mesh
+
+In a cluster-wide mesh, one `ServiceMeshControlPlane` (SMCP) watches all of the namespaces for an entire cluster. You can migrate an existing deployment from a multitenant mesh to a cluster-wide mesh using {SMProductName} version 2.5 or later. 
+
+[NOTE]
+====
+If a deployment must have more than one SMCP, then you cannot migrate the deployment to a cluster-wide mesh.
```

Comment:
What is meant by "deployment" here? When I see the word deployment, I usually think of the [Kubernetes CRD deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/), but that doesn't fit in this sentence.... so I'm not sure what this is saying.

Reply:
@longmuir basically, an existing service mesh. So if a customer has a mesh deployment and they must retain more than one SMCP, then migrating to a cluster-wide mesh is not an option for that customer. 

The published docs discusses deployment models: https://docs.openshift.com/container-platform/4.14/service_mesh/v2x/ossm-deployment-models.html

I think of a deployment as the customer has created and configured a mesh. To me, that's deployment in the context I'm trying to convey.

Reply:
Perhaps substitute the word "deployment" for "cluster"? More than one SMCP per cluster is clearer to me. The way deployment is used in this page, it could also potentially span multiple clusters...and I think the sentence really means more than one Istio control plane (which we call an SMCP in 2.x) per cluster.

Reply:
@longmuir thank you for the suggestion to use cluster instead of deployment to help clarify the context. I've updated the PR with this change.

Reply:
Looks good - thanks!